src/Form/atoms/FilterButton.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { ButtonHTMLAttributes} from 'react';
+import styled, { css } from 'styled-components';
+import { TypeButtonSizes } from '..';
+import { resetButtonStyles } from '../../common';
+import { dimensions } from '../../themes/common';
+import Icon, { IconWrapper } from '../../Icons/Icon';
+
+
+const StyledButton = styled.button<{size: TypeButtonSizes, fontSize: string,  isOpen?: boolean, hasFlipArrow?: boolean }>`
```

Comment:
I appreciate the forward thinking but we don't need any size variations at this point in time. As they're not styled to fully scale, it would be better to remove this and `fontSize` until we have a specific use case.